app/bundles/PageBundle/Model/RedirectModel.php

```diff
@@ -93,7 +87,7 @@ public function generateRedirectUrl(Redirect $redirect, $clickthrough = [], $sho
         }
 
         if ($shortenUrl) {
-            $url = $this->urlHelper->buildShortUrl($url);
+            $url = $this->shortener->shortenUrl($url);
```

Comment:
Isn't this used for email shorteners as well? I worry that this will break it for emails. When reading

> Only SMS messages will use the shortener service at the moment.

Reply:
No, just SMS. After the PR https://github.com/mautic/mautic/pull/12128 then we need update it

Reply:
@kuzmany I just merged https://github.com/mautic/mautic/pull/12128. Please rebase and update this so it can be merged ASAP before we release M5-beta next week.

Reply:
It's rebased. Also little refactored and add the option to enable/disable SMS to be clear about what channel shortener is supported.

<img width="579" alt="image" src="https://github.com/mautic/mautic/assets/462477/c4ff9f4d-3587-4cda-9162-b2f9af34c2b4">